src/FunderMaps.Data/Repositories/IncidentRepository.cs

```diff
@@ -266,7 +266,6 @@ UPDATE  report.incident
                             document_file = @document_file,
                             note = @note,
                             internal_note = @internal_note,
-                            contact = @email,
```

Comment:
Why was this removed?

Reply:
Contact is a foreign key, which can only be updated when the new entry exists. There should be no necessarily to update the contact link, but only ever the contact.